eclipse-sirius/sirius-components
# [1275] Add support for Label widget in Form representation

Fix #1275 

# Pull request template

## General purpose
What is the main goal of this pull request?

## Project management

## Architectural decision records (ADR)

## Dependencies

## Frontend

This section is not relevant if your contribution does not come with changes to the frontend.

### General purpose

### Typing
We need to improve the typing of our code, as such, we require every contribution to come with proper TypeScript typing for both changes contributing new files and those modifying existing files.
Please ensure that the following statements are true for each file created or modified (this may require you to improve code outside of your contribution).

- Hooks are properly typed:

## Backend

This section is not relevant if your contribution does not come with changes to the backend.

### General purpose

### Architecture

## Review

### How to test this PR?
[View.zip](https://github.com/eclipse-sirius/sirius-components/files/8949858/View.zip)

- Import the attached view dsl
- Create a new flow robot sample
- Create a new Form representation on the System root element
- You should see the label widget with "System" as title and "Name: Robot" as displayed value.
- Change the style of the Label description: you should see the changes in the Form editor (close and refresh the Form editor)
- Create a new conditional style (with 'aql:true' as condition) on the Label description: The conditional style should be displayed in the Form editor
